src/stores/OverlayStore/OverlayStore.ts

```diff
@@ -1179,10 +1237,6 @@ export class OverlayStore {
         return numGap + numHeight + this.defaultGap;
     }
 
-    @computed get base() {
```

Comment:
I prefer we set `OverlayStore.base` back to 5 instead of 1 to preserve the current paddings of the image view.